app/views/openid_connect/authorization/redirect.html.erb

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title><%= t('saml_idp.shared.saml_post_binding.redirecting') %> | <%= APP_NAME %></title>
+    <%= stylesheet_link_tag 'application', media: 'all' %>
+    <%= render_stylesheet_once_tags %>
```

Comment:
We could probably skip loading these?

Reply:
I'm thinking we'll want to render something other than a white page since a partner redirect may take some time?

Reply:
But we could add the stylesheets if/when we have content to be shown?

Reply:
Yeah, I was thinking of including something along the lines of `Redirecting to <a href="@oidc_redirect">` or similar?

Reply:
Ah, okay, I wasn't sure if you were planning on adding content as part of this pull request. If we do, makes sense to have the stylesheets 👍

Reply:
We could also leave that to a separate pull request? I guess I'm thinking we don't have anything shown currently for SAML, and we probably want to keep them in sync anyways? We could have UX folks from the team consider if there's a nicer way to communicate this.

Reply:
I think that makes sense, yeah.

Reply:
Spoke a little bit further, and since there may be content in the future and CSS should already be cached, we're going to leave the stylesheets in there for now.